Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -538,7 +538,7 @@ public IMixedRealityManager GetManager(Type type)
 
             if (manager == null)
             {
-                throw new NullReferenceException($"Unable to find {type.Name}.");
+                Debug.Log($"Unable to find {type.Name}.");
```

Comment:
Seems like this should be changed in the main branch.

Reply:
i anticipate this branch being merged into dev in the next day or so... it could go there, or just come along for the ride.

Reply:
Let's keep the PRs focused.

Reply:
this was needed for this pr as I was seeing exceptions. that is why it is here

Reply:
that said, I will be opening a pr into dev with this change and will handle the merge conflict prior to merging the feature branch there

Reply:
#2953 handles this exception and the others in the manager